ogcapi-features/README.md

```diff
@@ -122,6 +133,33 @@ Usually, the `self` link in a Collection must link to the same URL that was used
 it. However, implementations may choose to have the canonical location of the Collection be
 elsewhere. If this is done, it is recommended to include a `rel` of `canonical` to that location.
 
+Note that the `parent` link for a Collection should be point to the parent Catalog (such as the root Catalog, `/`) or Collection
+of that Collection, rather than the API sub-path of `/collections`.
+
+### Collection Items (/collections/{collectionId}/items)
+
+The following Link relations must exist in the ItemCollection object returned from the `/collections/{collectionId}/items` endpoint.
+
+| **rel**      | **href**                            | **Media Type**       | **From**  | **Description**      |
+| ------------ | ----------------------------------- | -------------------- | --------- | -------------------- |
+| `root`       | `/`                                 | application/json     | STAC Core | The root URI         |
+| `self`       | `/collections/{collectionId}/items` | application/geo+json | OAFeat    | Self reference       |
+| `collection` | `/collections/{collectionId}`       | application/json     | OAFeat    | Collection reference |
```

Comment:
This one keeps me a bit puzzled. Everywhere in STAC we use parent, but here it is collection. That's not very consistent. I'm wondering whether we should require parent and collection for consistency?

Reply:
The difference is that this is a link rel from the ItemCollection rather than an Item. The use of `collection` as the link rel is actually from OAFeat, since the Collection is the parent of all the Items, but *not* the parent of the ItemCollection (which has no parent).